src/lib/toastr/toast.component.ts

```diff
@@ -60,7 +60,8 @@ import { ToastrService } from './toastr.service';
 export class Toast implements OnDestroy {
   message?: string | SafeHtml | null;
   title?: string;
-  options: IndividualConfig;
+  options: GlobalConfig;
```

Comment:
@scttcper Is there a specific reason you chose IndividualConfig rather than GlobalConfig here?
Opening this up allows me to use global options in the template.

Reply:
The idea is that an individual toast shouldn't be concerned with what is supposed to happen globally. That's not its job. The `Toast` should only be concerned with what it itself needs to know and to also respond accordingly when something globally (e.g. `ToastrService`) tells it to do something.

Reply:
That's a valid point. I now followed the example of the **preventDuplicates** and **resetTimeoutOnDuplicate** flags and moved the global check into the ToastrService.